src/Core/src/HotReload/HotReloadHelper.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Maui.Hosting;
+
+namespace Microsoft.Maui.HotReload
+{
+	public static class MauiHotReloadHelper
```

Comment:
Does this needs to be public  ?

Reply:
It could be internal. The root views need to register themselves, as well as the IReplaceableView need to check if it was replaced. But both Comet and Controls need access to it.